transmart-rest-api/grails-app/controllers/org/transmartproject/interceptors/QueryAuditInterceptor.groovy

```diff
@@ -0,0 +1,57 @@
+package org.transmartproject.interceptors
+
+import groovy.transform.CompileStatic
+
+@CompileStatic
+class QueryAuditInterceptor extends AuditInterceptor {
```

Comment:
This class looks really high-maintainance...

Reply:
I reduced the message generation part, now it's as other interceptors